frontend/src/views/Course.vue

```diff
@@ -77,7 +77,19 @@ export default {
     this.$store.dispatch('course/getCourse', this.id)
     this.$store.dispatch('course/getQuestions', this.id)
     this.$store.dispatch('course/getReviews', this.id)
-  }
+  },
+  beforeRouteUpdate ({ params: { id }}, from, next) {
```

Comment:
I'm a genius for coming up with this

Reply:
'Something, something and that's why HTC was wrong in the Apple vs HTC case'